proto/opamp.proto

```diff
@@ -432,6 +444,18 @@ message TLSConnectionSettings {
   repeated string cipher_suites = 6;
 }
 
+// Status: [Development]
+message ProxyConnectionSettings {
+    // A URL, host:port or some other destination specifier.
+    string destination_endpoint = 1;
```

Comment:
The name `destination_endpoint` doesn't seem intuitive to me. Other ideas:
* `proxy_endpoint`
* `proxy_url`
* `endpoint`
* `url`